### PR TITLE
remove compare feature

### PR DESCRIPTION
**Pull request by alexandrasp**
remove from front-end dead code from compare feature. It was causing some bugs and will be
completely removed from kci front and back-end applications.

Signed-off-by: Alexandra Pereira <alexandra.pereira@collabora.com>

### CODE REVIEW

**Review by mgalka — APPROVED** [review]
LGTM

File: app/dashboard/static/js/common.js [hunk 1]
```diff
@@ -71,17 +73,14 @@ require.config({
...
             'app/view-boots-regressions': 'app/view-boots-regressions.2017.3.3',
-            'app/view-build-compare': 'app/view-build-compare.2017.4',
             'app/view-builds-all': 'app/view-builds-all.2017.5',
             'app/view-builds-id': 'app/view-builds-id.2017.4',
             'app/view-builds-job-branch-kernel': 'app/view-builds-job-branch-kernel.2017.7.1',
             'app/view-builds-job-kernel': 'app/view-builds-job-kernel.2017.4',
             'app/view-builds-job-kernel-defconfig': 'app/view-builds-job-kernel-defconfig.2017.3.3',
-            'app/view-builds-job-kernel-defconfig-logs': 'app/view-builds-job-kernel-defconfig-logs.2017.4',
-            'app/view-compare': 'app/view-compare.2017.3.3',
+            'app/view-builds-job-kernel-defconfig-logs': 'app/view-builds-job-kernel-defconfig-logs.2017.4',
```
Comment (gctucker) [inline app/dashboard/static/js/common.js]: looks like there's a new trailing whitespace here

**Review by gctucker — COMMENTED** [review]
Found a minor cosmetic thing, but LGTM otherwise.

File: app/dashboard/static/js/common.js (hunk 1 above)
Comment (alexandrasp) [reply app/dashboard/static/js/common.js]: solved!

**Comment by alexandrasp** [comment]
@gctucker I think we don't have a conflict here, right?

**Comment by gctucker** [comment]
OK I think it's conflicting with #63 so you can rebase when it's merged.

**Comment by gctucker** [comment]
@alexandrasp OK now there's a conflict to resolve :)

**Comment by alexandrasp** [comment]
> @alexandrasp OK now there's a conflict to resolve :)

@gtucker, it's solved.

**Comment by gctucker** [comment]
@alexandrasp Thanks, it's on staging :)

**Comment by gctucker** [comment]
Tested OK on staging.

**Pull request merged**